src/dotnet/devcontainer-template.json

```diff
@@ -12,16 +12,12 @@
             "description": ".NET version:",
             "proposals": [
                 "8.0",
-                "7.0",
                 "6.0",
```

Comment:
Can you cleanup from other .NET Templates as well? See https://github.com/search?q=repo%3Adevcontainers%2Ftemplates%20%2Fdotnet%3A&type=code